data-intel.txt

```diff
@@ -14,6 +14,7 @@
 2014     1324        662      18          68         145   2300       "Xeon E5-2699 v3"
 2015     1324        662      18          68         145   2300       "Xeon E5-2699 v3"
 2016     1548        774      22          77         145   2200       "Xeon E5-2699 v4"
-2017     4480       2240      28          120        205   2500       "Xeon Platinum 8180"
-2018     9320       4660      56          175        400   2600       "Xeon Platinum 9282"
-
+2017     4480       2240      28         120         205   2500       "Xeon Platinum 8180"
+2018     9320       4660      56         175         400   2600       "Xeon Platinum 9282"
+2020     5197       2598      28          96         250   2900       "Xeon Platinum 8380HL"
+2021     5888       2944      40         128         270   2300       "Xeon Platinum 8380"
```

Comment:
What's your source for this bandwidth number on 8380? It's 8-channel DDR4-3200, which would be a theoretical peak of 205 GB/s. (Of course it's much less in practice.)

Reply:
https://ark.intel.com/content/www/us/en/ark/products/212287/intel-xeon-platinum-8380-processor-60m-cache-2-30-ghz.html

And 8380-HL is 6-channel DDR4-3200, which is theoretical peak of 153.6 GB/s.
https://ark.intel.com/content/www/us/en/ark/products/205684/intel-xeon-platinum-8380hl-processor-38-5m-cache-2-90-ghz.html

Reply:
Oh yes, I'd applied a wrong formula inducted by precedents values for Xeon Platinum 9282 and Xeon Platinum 8180 inside this commit : https://github.com/karlrupp/cpu-gpu-mic-comparison/commit/90d3cc19bb5a19da28f3b0043c05e5701c548743
So, if the right formula is `Number_of_channels * Data_transfert_frequency * 8`,  It be better to fix also all bandwidth for 9282 according to  https://ark.intel.com/content/www/us/en/ark/products/194146/intel-xeon-platinum-9282-processor-77m-cache-2-60-ghz.html
and also for 8180 according to https://ark.intel.com/content/www/us/en/ark/products/120496/intel-xeon-platinum-8180-processor-38-5m-cache-2-50-ghz.html :

```
2017     4480       2240      28         128         205   2500       "Xeon Platinum 8180"
2018     9320       4660      56         175         400   2600       "Xeon Platinum 9282"
2020     5197       2598      28         153         250   2900       "Xeon Platinum 8380HL"
2021     5888       2944      40         205         270   2300       "Xeon Platinum 8380"
```

Reply:
Yes, I think those corrections should be applied.

Reply:
done here https://github.com/karlrupp/cpu-gpu-mic-comparison/pull/9/commits/3be915da7517f74d00d4e44ba6fe4c8340460cac